log-collector-script/linux/eks-log-collector.sh

```diff
@@ -20,7 +20,7 @@ export LANG="C"
 export LC_ALL="C"
 
 # Global options
-readonly PROGRAM_VERSION="0.6.2"
+readonly PROGRAM_VERSION="0.6.3"
```

Comment:
Didn't realize this thing even had versioning. I don't know what the semantics have been in the past, but this slew of changes seems to justify `0.7.0`?